editors/emacs/GNUmakefile

```diff
@@ -0,0 +1,18 @@
+CASK = ~/.cask/bin/cask
```

Comment:
What is cask? Is it a standard path? Do you document that it is necessary somewhere? (I don't have such a command on my machine.)

Reply:
It's standard for emacs development (see any emacs package, for instance, https://github.com/emacs-lsp/lsp-mode/blob/master/Makefile), but I shouldn't set this path